mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -126,28 +140,79 @@ def getOrientation(countryName):
         print("Error: Could not derive country extent from " + url)
         print("Exiting.")
         sys.exit(1)
-           
+
 
 def main(args):
     args = parser.parse_args()
-    cookbookFile = args.cookbookFile
-    layerPropertiesFile = args.layerConfig
+
+    # Construct a Crash Move Folder object if the cmf_description.json exists
     crashMoveFolder = args.crashMoveFolder
-    layerDirectory = args.layerDirectory
+    cmfFilePath = os.path.join(crashMoveFolder, "cmf_description.json")
```

Comment:
We could avoid hardcoding filenames here. For this tool, rather than asking the user for the path to the CMF directory itself, the argument could be for the `event_description.json`. I think it would be valid and appropriate for `event_description.json` to include a reference to the `cmf_description.json` file. This doesn't increase the number of arguments.